Intro_markdown.Rmd

```diff
@@ -232,31 +270,17 @@ There are other commands and various packages that can be used to import dataset
 
 ### Importing data from the Analytical Platform cloud storage (Amazon S3)
 
-Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. There are two options for packages that can be used to import data from S3: `Rs3tools` and `botor`.
```

Comment:
This section needs to be retained as Rs3tools is the recommended way in the AP user guide so I think we should be using this while also giving the botor equivalent as many new people will be inheriting code that uses botor.

Reply:
reinstated